Soomsil-USaint/Data/Client/RuSaint/GradeClient.swift

```diff
@@ -4,4 +4,253 @@
+            overallRank: Int,
+            overallStudentCount: Int,
+            lectures: [LectureDetail]?,
+            in context: NSManagedObjectContext
+        ) {
+            let semesterEntity = CDSemester(context: context)
+            semesterEntity.year = Int16(year)
+            semesterEntity.semester = semester
+            semesterEntity.gpa = gpa
+            semesterEntity.earnedCredit = earnedCredit
+            semesterEntity.semesterRank = Int16(semesterRank)
+            semesterEntity.semesterStudentCount = Int16(semesterStudentCount)
+            semesterEntity.overallRank = Int16(overallRank)
+            semesterEntity.overallStudentCount = Int16(overallStudentCount)
+            
+            let lectureEntities = lectures?.compactMap { lecture -> CDLecture? in
+                let cdLecture = CDLecture(context: context)
+                cdLecture.code = lecture.code
+                cdLecture.title = lecture.title
+                cdLecture.credit = Float(lecture.credit)
+                cdLecture.score = lecture.score
+                cdLecture.grade = lecture.grade.rawValue
+                cdLecture.professorName = lecture.professorName
+                return cdLecture
+            }
+            lectureEntities?.forEach { semesterEntity.addToLectures($0) }
+        }
+    }()
+    
+    static let previewVale: GradeClient = GradeClient {
+        [
+            Rusaint.SemesterGrade(year: 2024,
+                               semester: "2 학기",
+                               attemptedCredits: 2.0,
+                               earnedCredits: 2.0,
+                               pfEarnedCredits: 2.0,
+                               gradePointsAvarage: 0.0,
+                               gradePointsSum: 0.0,
+                               arithmeticMean: 0.0,
+                               semesterRank: Rusaint.UnsignedIntPair(first: 0, second: 0),
+                               generalRank: Rusaint.UnsignedIntPair(first: 12, second: 94),
+                               academicProbation: false,
+                               consult: false,
+                               flunked: false)
+        ]
+    } fetchGrades: { year, semester in
+        [
+            Rusaint.ClassGrade(year: "2024", semester: "2 학기", code: "", className: "", gradePoints: 0.0, score: .empty, rank: "", professor: "", detail: nil)
+        ]
+    } getAllSemesterGrades: {
+        [
+            CDSemester()
+        ]
+    } getGrades: { year, semester in
+        CDSemester()
+    } updateAllSemesterGrades: { rusaintSemesterGrades in
+        return
+    } updateGrades: { year, semester, newLectures in
+        return
+    } updateGPA: { year, semester, gpa in
+        return
+    } addGrades: { newSemester in
+        return
+    } deleteAllSemesterGrades: {
+        return
+    } deleteGrades: { year, semester in
+        return
+    }
+
+    static let testValue: GradeClient = previewVale
```

Comment:
```suggestion
    static let testValue: GradeClient = previewValue
```
앞서 남긴 리뷰로 여기도 바뀌어야 할 것 같아요!